components/ApplicationsList/ApplicationsList.jsx

```diff
@@ -179,30 +183,41 @@ const ApplicationsList = ({
         </a>
       </p>
       <p>
-        <a
-          href="#"
+        <p>
```

Comment:
I wonder if to just help separate these links for Miranda (and others) we could add a header (Export payment details) and that'll add a bit of space between the links. What do you think?

Reply:
Is this a header before all the buttons or before each individual button?

Reply:
One header for the three buttons, so:

### Export payment details
<button>
<button>
<button>